pkg/trivy/wrapper.go

```diff
@@ -14,14 +13,20 @@ import (
 	"strings"
 )
 
+type ImageRef struct {
+	Name     string
+	Auth     RegistryAuth
+	Insecure bool
+}
+
 // RegistryAuth wraps registry credentials.
 type RegistryAuth struct {
 	Username string
 	Password string
 }
 
 type Wrapper interface {
-	Run(imageRef string, auth RegistryAuth, insecureRegistry bool) (ScanReport, error)
+	Scan(imageRef ImageRef) (ScanReport, error)
```

Comment:
Is anyone or any other call-site using the old interface besides within this project? Would this break anyone/anything?

Reply:
> Is anyone or any other call-site using the old interface besides within this project? Would this break anyone/anything?

This is our internal interface. We can break and refactor as needed